composer.json

```diff
@@ -39,6 +39,10 @@
         {
             "type": "git",
             "url":  "https://github.com/eileenmcnaughton/omnipay-tests.git"
+        },
+        {
```

Comment:
I actually did make this change in master

Reply:
I think composer update still needs to be run and committed in master though?